repos/fdbt-site/src/pages/api/reuseOperatorGroup.ts

```diff
@@ -25,12 +28,37 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
 
         const multipleOperators = await getOperatorGroupByNocAndId(selectedOperatorGroup, noc);
 
+        const ticket = getSessionAttribute(req, MATCHING_JSON_ATTRIBUTE);
+        const matchingJsonMetaData = getSessionAttribute(req, MATCHING_JSON_META_DATA_ATTRIBUTE);
+
         if (multipleOperators) {
-            updateSessionAttribute(req, MULTIPLE_OPERATOR_ATTRIBUTE, {
-                selectedOperators: multipleOperators.operators,
-                id: multipleOperators.id,
-            });
-            updateSessionAttribute(req, REUSE_OPERATOR_GROUP_ATTRIBUTE, []);
+            if (ticket && matchingJsonMetaData) {
+                // edit mode
+                const updatedTicket = {
+                    ...ticket,
+                    additionalNocs: multipleOperators.operators.map((operator) => operator.nocCode),
+                };
+
+                // put the now updated matching json into s3
+                // overriding the existing object
+                await putUserDataInProductsBucketWithFilePath(updatedTicket, matchingJsonMetaData.matchingJsonLink);
+
+                updateSessionAttribute(req, MULTIPLE_OPERATOR_ATTRIBUTE, undefined);
+
+                redirectTo(
+                    res,
+                    `/products/productDetails?productId=${matchingJsonMetaData?.productId}${
+                        matchingJsonMetaData.serviceId ? `&serviceId=${matchingJsonMetaData?.serviceId}` : ''
```

Comment:
We dont need the serviceId stuff here because we know that the ticket is a multiOperatorGeoZone, and as such isnt associated with one service.